CHANGELOG.rst

```diff
@@ -7,6 +7,11 @@ CHANGELOG
 
 * feature: Estimators: add support for TensorFlow-1.5.0
 * feature: Estimators: add support for MXNet-1.0.0
+* feature: Tests: use ``sagemaker_timestamp`` when creating endpoint names in integration tests
```

Comment:
cli?

Reply:
didn't notice it hadn't been added before! just added it to where it should've been